_opensearch/rest-api/nodes-apis/nodes-reload-secure.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: default
+title: Nodes reload secure settings
+parent: Nodes APIs
+grand_parent: REST API reference
+nav_order: 50
+---
+
+# Nodes reload secure settings
+
+The nodes reload secure settings endpoint allows you to change secure settings on a node and reload the secure settings without restarting the node.
+
+## Path and HTTP methods
+
+```
+POST _nodes/reload_secure_settings
+POST _nodes/<nodeId>/reload_secure_settings
+```
+
+## Path parameter
+
+You can include the following optional path parameter in your request:
+
+Parameter | Type | Description
+:--- | :--- | :---
+nodeId | String | A comma-separated list of nodeIds to filter results. Supports [node filters]({{site.url}}{{site.baseurl}}/opensearch/rest-api/nodes-apis/index/#node-filters). Defaults to `_all`.
```

Comment:
Change to "A comma-separated list of nodeIds used to filter results" (add "used"). In the next file, we use "node IDs" (instead of "nodeIds"). Let's make consistent.